packages/codegen.go/src/time.ts

```diff
@@ -181,17 +185,33 @@ func (t dateTimeRFC3339) MarshalText() ([]byte, error) {
 }
 
 func (t *dateTimeRFC3339) UnmarshalJSON(data []byte) error {
-	layout := utcDateTimeJSON
-	if tzOffsetRegex.Match(data) {
+	tzOffset := tzOffsetRegex.Match(data)
```

Comment:
How much of perf hit do we take b/c of this change?

Reply:
No clue, but I doubt it's significant given we're already O(N^2) due to nested called to `json.Unmarshal` :(